.github/workflows/ci.yaml

```diff
@@ -9,73 +9,68 @@ on:
   pull_request:
 
 jobs:
-  test:
-    strategy:
-      matrix:
-        python-version: [3.7]
+  lint:
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+      - name: install pre-commit
+        run: |
+          python -m pip install -U pip setuptools wheel
+          python -m pip install pre-commit
+      - name: run pre-commit
+        run: pre-commit run -a
 
+  test-sdk:
+    runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Get latest pip version
-      run: |
-        python -m pip install --upgrade pip setuptools wheel
-    - name: Lint
-      run: |
-        pip install pre-commit
-        pre-commit run -a
-    - name: Install dependencies for funcx-sdk
-      run: |
-        python -m pip install -r funcx_sdk/requirements.txt
-        python -m pip install -r funcx_sdk/test-requirements.txt
-        pip list
-    - name: Check for vulnerabilities in libraries
-      run: |
-        pip install safety
-        pip freeze | safety check
-    - name: Test sdk by just importing
-      run: |
-        cd funcx_sdk
-        pip install .
-        python -c "from funcx.sdk.client import FuncXClient"
-        cd ..
-#    - name: Test with pytest
-#      run: |
-#        pytest
-    - name: Install dependencies for funcx-endpoint
-      run: |
-        python -m pip install -r funcx_endpoint/requirements.txt
-        python -m pip install -r funcx_endpoint/test-requirements.txt
-        pip list
-    - name: Check for vulnerabilities in libraries
-      run: |
-        pip install safety
-        pip freeze | safety check
-    - name: Test funcx-endpoint by just importing
-      run: |
-        cd funcx_endpoint
-        pip install .
-        python -c "from funcx_endpoint.version import VERSION"
-        funcx-endpoint -v
-        cd ..
-    - name: Lint with Flake8
-      run: |
-        flake8 funcx_endpoint
-    - name: Test with pytest
-      run: |
-        PYTHONPATH=funcx_endpoint python -m coverage run -m  pytest funcx_endpoint/tests/funcx_endpoint
-    - name: Report coverage with Codecov
-      run: |
-        codecov --token=${{ secrets.CODECOV_TOKEN }}
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: install requirements
+        run: |
+          python -m pip install -U pip setuptools wheel
+          python -m pip install './funcx_sdk[test]'
+          pip install safety
+      - name: run safety check
+        run: safety check
```

Comment:
previously this had the output of pip freeze piped in - this comment is to check that its fine to have removed that?

Reply:
Yeah, piping in pip freeze does literally nothing as far as I know. safety operates on it's current venv -- I use it this way in other projects and it works.